Doc/library/http.client.rst

```diff
@@ -477,6 +485,18 @@ statement.
 
    Is ``True`` if the stream is closed.
 
+.. method:: HTTPResponse.geturl()
+
+   Returns the URL of the resource retrieved. Recommended to use :attr:`~HTTPResponse.url` instead.
```

Comment:
Maybe this could be condensed to:

```
Deprecated method equivalent to :attr:`HTTPResponse.url`
```

Reply:
Is that okay to call it "deprecated" even when it doesn't return any DeprecationWarnings when used?

Reply:
Yes, we name that «doc-deprecated», it’s a useful degree of deprecation.
(i.e. don’t break code that is fine, don’t annoy with warnings, just recommend the obvious way to do it in docs for people who want that)

Reply:
Okay, I've went ahead and called it "deprecated" in the documentation. How do you keep track of which things mentioned in the documentation are deprecated and which are doc-deprecated?